x500/cert/acme/src/test/java/org/wildfly/security/x500/cert/acme/AcmeClientSpiTest.java

```diff
@@ -261,32 +261,32 @@ public void testGetNonce() throws Exception {
     @Test
     public void testObtainCertificateChain() throws Exception {
         server = setupTestObtainCertificate();
-        AcmeAccount account = populateAccount(ACCOUNT_1);
-        String domainName = "fjsljghasldfjgk.com"; // randomly generated domain name
+        AcmeAccount account = populateAccount(ACCOUNT_3_V2);
+        String domainName = "fjsljghasldfjgkv2.com"; // randomly generated domain name
         obtainCertificateChain(null, -1, account, domainName);
     }
 
     @Test
     public void testObtainCertificateChainWithKeySize() throws Exception {
         server = setupTestObtainCertificateWithKeySize();
-        AcmeAccount account = populateAccount(ACCOUNT_6);
-        String domainName = "inlneseppwkfwew.com"; // randomly generated domain name
+        AcmeAccount account = populateAccount(ACCOUNT_6_V2);
+        String domainName = "inlneseppwkfwewv2.com"; // randomly generated domain name
         obtainCertificateChain("RSA", 4096, account, domainName);
     }
 
     @Test
     public void testObtainCertificateChainWithECPublicKey() throws Exception {
         server = setupTestObtainCertificateWithECPublicKey();
-        AcmeAccount account = populateAccount(ACCOUNT_7);
-        String domainName = "mndelkdnbcilohg.com"; // randomly generated domain name
+        AcmeAccount account = populateAccount(ACCOUNT_7_V2);
+        String domainName = "mndelkdnbcilohgv2.com"; // randomly generated domain name
         obtainCertificateChain("EC", 256, account, domainName);
     }
 
     @Test
     public void testObtainCertificateChainWithUnsupportedPublicKey() throws Exception {
         try {
             server = setupTestObtainCertificateWithUnsupportedPublicKey();
-            AcmeAccount account = populateAccount(ACCOUNT_7);
+            AcmeAccount account = populateAccount(ACCOUNT_7_V2);
```

Comment:
@ashley-abdelsayed98 Please double check the setup method for this test. Just tried running it locally to see the message that it fails with and it seems to be "Parameter 'nonce' may not be null" instead of a message related to the key being unsupported.

Reply:
Please also check the corresponding test case in https://github.com/wildfly/wildfly-core/pull/3868.

Reply:
Fixed